flight-service/src/test/resources/application-test.yml

```
@@ -1,42 +1,53 @@
-      defaultZone: http://localhost:19090/eureka/
+    enabled: false
 
 logging:
   level:
-    org:
-      hibernate:
-        SQL: debug
-        type: trace
\ No newline at end of file
+    org.hibernate.SQL: debug
+    org.hibernate.type.descriptor.sql.BasicBinder: trace
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)